### DISCUSSION TITLE
Welcome to LGBTQIA-in-China Discussions!

### DISCUSSION THREAD

**Discussion by kecrily** [LGBT-CN/LGBTQIA-In-China#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KevinZonda** [comment]
Cool!
Hi everyone! I'm Kevin from China. You can also call me Huihui. Now, I am a student who is learning computing.

**Comment by ghost** [comment]
do not bring virus to CN，just enjoying it in your own country is OK, OK?

**Reply by KevinZonda** [reply]
?

**Reply by xiaojing5576** [reply]
> ?

hanpi

**Reply by KevinZonda** [reply]
> > ?
> 
> hanpi

Oh mate. Look at your bleedy word, it's sublimely ELEGANT, isn't it?

**Comment by xiaojing5576** [comment]
Not only in China, but also in other countries,there is malice in targeting China alone

**Reply by KevinZonda** [reply]
Cool idea. There are so many people in the world who commit crimes, is it too much of a target to imprison only some of them?

**Reply by KevinZonda** [reply]
At the same time, we are not obliged to interfere in the internal affairs of other countries. Please be clear about the responsibilities. Thanks.
